install-cockroachdb.md

```diff
@@ -417,32 +424,38 @@ $(document).ready(function(){
 
+{{site.data.alerts.callout_danger}}Running a stateful application like CockroachDB in Docker is more complex and error-prone than most uses of Docker and is not recommended for production deployments. To run a physically distributed cluster in containers, use an orchestration tool like Kubernetes or Docker Swarm. See <a href="orchestration.html">Orchestration</a> for more details.{{site.data.alerts.end}}
+
 <ol>
   <li>
-    <p>Install <a href="https://docs.docker.com/docker-for-windows/">Docker for Windows</a>.</p>
+    <p>Install <a href="https://docs.docker.com/docker-for-windows/">Docker for Windows</a>. Please carefully check that you meet all prerequisites.</p> 
   </li>
   <li>
-    <p>Confirm that the Docker daemon is running in the background:</p>
+    <p>Open PowerShell and confirm that the Docker daemon is running in the background:</p>
+
+    <div class="language-powershell highlighter-rouge"><pre class="highlight"><code><span class="nb">PS </span>C:<span class="se">\U</span>sers<span class="se">\u</span>sername&gt; docker version</code></pre></div>
 
-    <div class="highlighter-rouge"><pre class="highlight"><code>$ docker version</code></pre>
-    </div>
     <p>If you don't see the server listed, start the <strong>Docker</strong> application.</p>
   </li>
   <li>
     <p>Pull the official CockroachDB image from <a href="https://hub.docker.com/r/cockroachdb/cockroach/" data-eventcategory="win-docker-step3">Docker Hub</a>:</p>
 
-    <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="win-docker-step3"><span class="gp" data-eventcategory="win-docker-step3">$ </span>docker pull cockroachdb/cockroach:{{site.data.strings.version}}</code></pre>
-    </div>
+    <div class="language-powershell highlighter-rouge"><pre class="highlight"><code data-eventcategory="win-docker-step3"><span class="nb" data-eventcategory="win-docker-step3">PS </span>C:<span class="se">\U</span>sers<span class="se">\u</span>sername&gt; docker pull cockroachdb/cockroach:{{site.data.strings.version}}</code></pre></div>
   </li>
   <li>
       <p>Make sure CockroachDB installed successfully:</p>
 
-      <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="win-docker-step4"><span class="gp" data-eventcategory="win-docker-step4">$ </span>docker run --rm cockroachdb/cockroach:{{site.data.strings.version}} version</code></pre></div>
+      <div class="language-powershell highlighter-rouge"><pre class="highlight"><code data-eventcategory="win-docker-step4"><span class="nb" data-eventcategory="win-docker-step4">PS </span>C:<span class="se">\U</span>sers<span class="se">\u</span>sername&gt; docker run --rm cockroachdb/cockroach:{{site.data.strings.version}} version</code></pre></div>
+
   </li>
   <li>
     <p>Get future release notes emailed to you:</p>
```

Comment:
This shows up as blank for me.  Perhaps it's due to hubspot any my tracker blocking.